<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorSelectionDialog</class>
 <widget class="QDialog" name="DoctorSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="departmentsListWidget">
       <property name="maximumSize">
        <size>
         <width>16777213</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="doctorsTableWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
